packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookieBanner-legacy.component.js

```diff
@@ -0,0 +1,32 @@
+const Page = require('@justeat/f-wdio-utils/src/page.object');
+
+module.exports = class CookieBanner extends Page {
+
+    get component () { return $('[data-test-id="legacyCookieBanner-component"]')}
+    get cookiePolicyLink () { return this.component.$('[data-test-id="cookie-policy-link"]')}
+    get closeButton () { return this.component.$('[data-test-id="cookieBanner-close-button"]')}
+
+    open(tenant = 'en-gb') {
+        super.openComponent('organism', `cookie-banner-component&knob-Locale=en-${tenant.toUpperCase()}`);
+    }
+
+    waitForComponent() {
+        this.component.waitForExist();
+    }
+
+    isCookieBannerComponentDisplayed() {
+        return this.component.isDisplayed();
+    }
+
+    clickCookiePolicyLink() {
+        this.cookiePolicyLink.click();
+    }
+
+    close() {
```

Comment:
qq - do you need to `return this` here?

Reply:
We don't need it for here as we're just doing a click function on this.